babel/board.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 import * as _ from 'lodash';
+let SortedSet = require('collections/sorted-set');
```

Comment:
I saw that but wasn't sure. I was looking through the collections module and in order to use `union()` it had to be a SortedSet and not a Set.